app/src/main/java/com/google/samples/apps/sunflower/GardenActivity.kt

```diff
@@ -60,12 +60,11 @@ class GardenActivity : AppCompatActivity() {
             binding.drawerLayout.addDrawerListener(it)
         }
 
-        binding.navigationView.setupWithNavController(
-            Navigation.findNavController(
-                this,
-                R.id.garden_nav_fragment
-            )
+        val navController = Navigation.findNavController(
```

Comment:
This can be one line

Reply:
@tiembo I've formatted the line, plz check it out.